### PR TITLE
add SDK docs

### PR DESCRIPTION
**Pull request by justinzhou93**

### CODE REVIEW

**Comment by justinzhou93** [comment]
worth spinning up locally and seeing how it looks 
image

I put almost all of the documentation as comments with the function definitions, since they all live inside of a class function. It seemed more easily understandable this way, but lemme know this is too confusing.

File: _docs/automation-time-js-sdk.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+** We are still working on the RPC call to generate the task_id. Please save your `provided_id` so you can generate this later.
+
+#### Recurrer Class
+```javascript
+/**
+ * Recurring timestamps is a feature that allows for users to schedule regularly
+ * recurring tasks with a single blockchain extrinsic, rather than through multiple
+ * extrinsic calls, as evidenced by the array of timestamps accepted in the Scheduler
+ * class. This is currently only limited to 24 instances. These timstamps do not
```
Comment (irsal) [inline _docs/automation-time-js-sdk.md:212]: spelling: timestamps

File: _docs/automation-time-js-sdk.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+ * have to be evenly spaced. It can be any timestamps in the future, on the hour, and 
+ * limited to up to 24 occurences per extrinsic call.
```
Comment (irsal) [inline _docs/automation-time-js-sdk.md:214]: spelling: occurrences

**Review by irsal — CHANGES_REQUESTED** [review]
We definitely need cookbooks, or something like this https://polkadot.js.org/docs/api/cookbook/blocks

I'd go through the code snippet for building an hourly call with validation and error handling

File: _docs/automation-time-js-sdk.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
+---
+title: JS SDK - Time Triggers
+subtitle: The API specifications for Time Triggers
+author: justin
+tags: [api, time, triggers, js, sdk]
+---
+
+This is the documentation for the Javascript SDK for the Time Trigger blockchain APIs. You can find the SDK here: https://github.com/OAK-Foundation/OAK-JS-SDK.
```
Comment (irsal) [inline _docs/automation-time-js-sdk.md:8]: use mark down link notation `[](url)`

File: _docs/automation-time-js-sdk.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+
+NOTE: the below is just an example, but we do not recommend creating new instances of the Scheduler, Recurrer, etc classes on each invocation. It's probably best to start your service with the classes instantiated and then utilize those classes as needed on an ongoing basis.
+
+```javascript
+async function NEUNotify(senderAddress) {
+  await web3Enable('neumann-demo');
+  const injector = await web3FromAddress(senderAddress);
+  const scheduler = new Scheduler(oakConstants.OakChains.NEU)
+  const recurrer = new Recurrer()
+  const timestamps = recurrer.getHourlyRecurringTimestamps(Date.now(), 24)
```
Comment (irsal) [inline _docs/automation-time-js-sdk.md:296]: Add a comment above this explaining what the intention is

File: _docs/automation-time-js-sdk.md [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+  const providedID = uuid.v4()
```
Comment (irsal) [inline _docs/automation-time-js-sdk.md:297]: add a comment recommending that this gets savedsomewhere

**Review by irsal — APPROVED** [review]
so much better; a few non-blocking items to edit

**Pull request merged**